app/renderer/components/bookmarks/addEditBookmarkHanger.js

```diff
@@ -160,8 +161,8 @@ class AddEditBookmarkHanger extends React.Component {
 
   mergeProps (state, ownProps) {
     const currentWindow = state.get('currentWindow')
-    const bookmarkDetail = currentWindow.get('bookmarkDetail')
-    const currentDetail = bookmarkDetail.get('currentDetail')
+    const bookmarkDetail = currentWindow.get('bookmarkDetail') || new Immutable.Map()
```

Comment:
Let's do it like this, which is more reliable. Same for `currentDetail`.

`currentWindow.get('bookmarkDetail', Immutable.Map())`